contents/customers/zealot.md

```diff
@@ -0,0 +1,84 @@
+logoDark: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/zealot_logo_d_f47cccf783.png
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/zealot_posthog_error_tracking_14fb1ab6cb.jpg
+industries:
+  - Recruitment
+users:
+  - Engineering
+  - Leadership
+  - Founder
+toolsUsed:
+  - LLM observability
+  - Session recording
+  - Error tracking
+  - Product analytics
+date: 2023-09-15
+---
+
+As CTO and co-founder of Zealot, an AI-powered customer activation platform, Brandon Jakobson often sweats the details of product improvement. He’s constantly thinking how the team can scale faster, ship more, and do more for users. 
+
+“I’m deeply passionate about it and I have a lot of ideas for the future,” says Brandon. “How can we make sure our systems are always up, and that we’re always getting faster? Truthfully, it keeps me awake at night.”
+
+Knowing product analytics are essential for making such decisions, Brandon initially installed both Amplitude and PostHog side-by-side. PostHog had initially been personally recommended, whereas Amplitude was the traditional tool for many of the large enterprises that use Zealot’s software. 
+
+“Our customers are enterprises and they all use Amplitude, but I personally just love PostHog,” says Brandon. “One big difference is that when I’ve needed help with PostHog I get through to people like [David](https://posthog.com/community/profiles/30203) — the actual engineers building the tools, not a customer support agent.”
+
+“Plus, [Amplitude](/blog/posthog-vs-amplitude) was very clunky and it was missing data points. We would notice it was having a hard time capturing some things when we used it in parallel with PostHog. Using PostHog just became second nature and we felt it was definitely exceeding Amplitude.”
+
+As a result the team quickly doubled down on PostHog and Brandon immediately started inviting more of the team to use it. 
+
+<BorderWrapper>
+<Quote
+    imageSource="/images/customers/brandon.png"
+    size="md"
+    name="Brandon Jakobson"
+    title="Co-founder & CTO at Zealot"
+    quote={`“I’m so glad you guys don’t price based on seats. As soon as I realized that, I invited my whole team.”`}
+/>
+</BorderWrapper>
+
+## Switching from BugSnag to PostHog
+
+As Brandon’s team got more familiar with PostHog, they steadily uncovered new features and how it could simplify their stack even further. Again, a personal recommendation helped Brandon discover a key feature he now uses every day — [error tracking](/error-tracking). 
+
+Initially in beta, error tracking has since been released into general availability — and Brandon’s feedback even helped shape the eventual release despite the fact that Zealot was initially using BugSnag. 
+
+“BugSnag was bought by a company called Smartbear and...their tooling was just really old,” explains Brandon. “Like, really old. Ancient, and ugly. But we put up with it because it did the job and we could get notifications about new errors.”
```

Comment:
```suggestion
“BugSnag was bought by a company called Smartbear and... their tooling was just really old,” explains Brandon. “Like, really old. Ancient, and ugly. But we put up with it because it did the job and we could get notifications about new errors.”
```